web-external/src/components/dialog/file-selector/body.jsx

```diff
@@ -1,18 +1,29 @@
 import React from 'react';
-import jQuery from 'jquery';
+import $ from 'jquery';
 
 import { setFileNavigation } from '../../../actions';
-
 import RootSelector from '../../common/root-selector';
 
+/* hacked to remove the download and view links */
+/* TODO(opadron): flags controlling these links should be upstreamed */
```

Comment:
Can we remove the TODO from the comment then? Or maybe change it to a reference to the upstream fix?